### PR TITLE
Clear resource handlers at runtime

### PR DESCRIPTION
**Pull request by Fylax**
Added Routes::Clear<Verb> and router.clear_verb to remove all handlers on a given resource for a given verb

### CODE REVIEW

File: include/pistache/router.h [hunk 1]
```diff
@@ -154,12 +158,20 @@ class Router {
     void del(std::string resource, Route::Handler handler);
     void options(std::string resource, Route::Handler handler);
 
+    void clear_get(std::string resource);
```
Comment (oktal) [inline include/pistache/router.h]: Pistache uses camel case. Should be `clearGet`

File: include/pistache/router.h [hunk 2]
```diff
@@ -154,12 +158,20 @@ class Router {
...
+    void clear_post(std::string resource);
```
Comment (oktal) [inline include/pistache/router.h]: See above

File: src/server/router.cc [hunk 3]
```diff
@@ -321,6 +351,18 @@ Router::addRoute(Http::Method method, std::string resource, Route::Handler handl
     r.push_back(Route(std::move(resource), method, std::move(handler)));
 }
 
+void Router::clearRoutes(Http::Method method, std::string resource) {
+    auto& r = routes[method];
+    for(int i = 0; i < r.size(); i++) {
```
Comment (oktal) [inline src/server/router.cc]: `i` sould be `size_t`

**Comment by Fylax** [comment]
I'll fix it as soon as possible

**Pull request closed without merging**